### PR TITLE
Updated Cookies & Caching section

### PR DESCRIPTION
**Pull request by lukey78**
Correct code for Varnish 4 (unset instead of remove)

### CODE REVIEW

File: cookbook/cache/varnish.rst [hunk 1]
```diff
@@ -76,7 +76,9 @@ session cookie, if there is one, and get rid of all other cookies so that pages
 are cached if there is no active session. Unless you changed the default
 configuration of PHP, your session cookie has the name ``PHPSESSID``:
 
-.. code-block:: varnish4
+.. configuration-block::
+
+    .. code-block:: varnish4
 
     sub vcl_recv {
```
Comment (xabbuh) [inline cookbook/cache/varnish.rst]: The code block itself now needs to be indented by four spaces.

**Comment by javiereguiluz** [comment]
This pull request looks ready to be merged.

File: cookbook/cache/varnish.rst [hunk 2]
```diff
@@ -76,23 +76,44 @@ session cookie, if there is one, and get rid of all other cookies so that pages
...
+                set req.http.Cookie = regsuball(req.http.Cookie, ";[^ ][^;]*", "");
+                set req.http.Cookie = regsuball(req.http.Cookie, "^[; ]+|[; ]+$", "");
+    
+                if (req.http.Cookie == "") {
+                    // If there are no more cookies, remove the header to get page cached.
+                    remove req.http.Cookie;
+                }
+            }
+        }
+
```
Comment (xabbuh) [inline cookbook/cache/varnish.rst]: This blank line should be removed.

**Comment by xabbuh** [comment]
@lukey78 I just left you one last minor comment. Sorry for reacting so lately, but unfortunately we are not notified when someone pushes additional commits to a pull request.

@dbu Would you mind having a look at this change please? You do have much more expertise with Varnish than me. :)

**Comment by dbu** [comment]
according to the [varnish changelog](https://www.varnish-cache.org/trac/browser/doc/changes.rst) remove was indeed replaced by unset. the wiki page where this [example comes from](https://www.varnish-cache.org/trac/wiki/VCLExampleRemovingSomeCookies) seems to not be updated...

**Comment by javiereguiluz** [comment]
This PR can be labeled as finished because @dbu verified the proposed changes and the reamining blank page can be removed during the merge process. Thanks

**Comment by xabbuh** [comment]
Thank you for the changes @lukey78 and please apologise the slow merge. I squashed your commits so that's the reason the pull request is shown as closed. I also removed the additional blank line in ca3b830.

@dbu Thanks for reviewing!

**Pull request closed without merging**